.github/workflows/action-test.yml

```
@@ -46,46 +47,45 @@ jobs:
-#    steps:
-#      - uses: actions/checkout@v2
-#      - uses: conda-incubator/setup-miniconda@v2
-#        with:
-#          activate-environment: esmvaltool
-#          environment-file: environment.yml
-#          python-version: ${{ matrix.python-version }}
-#          miniconda-version: "latest"
-#          channels: conda-forge
-#      - shell: bash -l {0}
-#        run: mkdir -p test_osx_artifacts_python_${{ matrix.python-version }}
-#      - shell: bash -l {0}
-#        run: conda --version 2>&1 | tee test_osx_artifacts_python_${{ matrix.python-version }}/conda_version.txt
-#      - shell: bash -l {0}
-#        run: python -V 2>&1 | tee test_osx_artifacts_python_${{ matrix.python-version }}/python_version.txt
-#      - shell: bash -l {0}
-#        run: pip install -e .[develop] --use-feature=2020-resolver 2>&1 | tee test_osx_artifacts_python_${{ matrix.python-version }}/install.txt
-#      # ncurses needs to be from conda-forge and not main channel
-#      # for now it's turned off since we're not testing R/Julia installs
-#      # - shell: bash -l {0}
-#      #   run: conda install -y conda-forge::ncurses 2>&1 | tee test_osx_artifacts_python_${{ matrix.python-version }}/ncurses.txt
-#      # - shell: bash -l {0}
-#      #   run: esmvaltool install R
-#      # - shell: bash -l {0}
-#      #   run: esmvaltool install Julia
-#      - shell: bash -l {0}
-#        run: pytest -n 2 -m "not installation" 2>&1 | tee test_osx_artifacts_python_${{ matrix.python-version }}/test_report.txt
-#      - name: Upload artifacts
-#        if: ${{ always() }}  # upload artifacts even if fail
-#        uses: actions/upload-artifact@v2
-#        with:
-#          name: Test_OSX_python_${{ matrix.python-version }}
-#          path: test_osx_artifacts_python_${{ matrix.python-version }}
+  osx:
+    runs-on: "macos-latest"
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+      fail-fast: false
+    name: OSX Python ${{ matrix.python-version }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          activate-environment: esmvaltool
+          environment-file: environment_osx.yml
+          python-version: ${{ matrix.python-version }}
+          miniconda-version: "latest"
+          channels: conda-forge
+      - name: Install libomp with homebrew
+        run: brew install libomp
+      - shell: bash -l {0}
+        run: mkdir -p test_osx_artifacts_python_${{ matrix.python-version }}
+      - shell: bash -l {0}
+        run: conda --version 2>&1 | tee test_osx_artifacts_python_${{ matrix.python-version }}/conda_version.txt
+      - shell: bash -l {0}
+        run: python -V 2>&1 | tee test_osx_artifacts_python_${{ matrix.python-version }}/python_version.txt
+      - shell: bash -l {0}
+        run: pip install -e .[develop] --use-feature=2020-resolver 2>&1 | tee test_osx_artifacts_python_${{ matrix.python-version }}/install.txt
```

This is the default with the latest version of pip, so I think we do not need this anymore.

```suggestion
        run: pip install -e .[develop] 2>&1 | tee test_osx_artifacts_python_${{ matrix.python-version }}/install.txt
```